vulnerable_people_form/form_pages/shared/location_tier.py

```diff
@@ -47,12 +50,14 @@ def update_location_tier_by_postcode(postcode, app):
 
         lad_code = get_ladcode_from_postcode(postcode)
         if app.shielding_advice.is_la_shielding(lad_code):
-            set_shielding_advice(1)
+            set_shielding_advice(ShieldingAdvice.ADVISED_TO_SHIELD.value)
         else:
-            set_shielding_advice(0)
+            set_shielding_advice(ShieldingAdvice.NOT_ADVISED_TO_SHIELD.value)
 
         if postcode in app.config["POSTCODE_TIER_OVERRIDE"]:
```

Comment:
can this be set on the app it self in init.py and not reference the config property every-time as this logic is in quite a few places

Reply:
Done